api/python/provisioner/commands/deploy_dual.py

```diff
@@ -189,10 +189,18 @@ def _run_states(self, states_group: str, run_args: run_args_type):
                     self._apply_state(
                         f"components.{state}", secondaries, stages
                     )
+                elif state in (
+                    "ha.cortx-ha",
+                ):
+                    # Execute first on primary then on secondary sequentially.
+                    nodes = Deploy._get_node_list()
+                    self._apply_state(f"components.{state}", primary, stages)
+                    if primary in nodes:
+                        nodes.remove(primary)
+                    for node in nodes:
+                        self._apply_state(f"components.{state}", node, stages)
                 else:
                     self._apply_state(f"components.{state}", targets, stages)
-                    if state == "ha.cortx-ha":
```

Comment:
not sure how that condition can be True but please double-check if we need to ensure consul in scope of cortx-ha setup

Reply:
`check_consul_running` was specially written for dual node and It has old consul service names now consul service names are changed. That's why I have removed. If we required this check we can add this check in separate PR